daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/BlindingInfo.scala

```diff
@@ -10,8 +10,9 @@ import com.daml.lf.value.Value.ContractId
 /** A transaction's blinding information, consisting of disclosure and
   * divulgence info.
   *
-  * "Disclosure" tells us which transaction nodes to communicate to which
-  * parties.
+  * "Disclosure" tells us which transaction actions to communicate to which
```

Comment:
I wonder if blinding info should also store information about which nodes have been rolled back or if that’s better left to something else. Either way, I’d postpone that decision until we get to the usesites of blinding info. For the scenario service usecase, we only need this.